libparsec/crates/protocol/tests/authenticated_cmds/v4/message_get.rs

```diff
@@ -1,15 +1,77 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+use libparsec_tests_lite::hex;
+
 use super::authenticated_cmds;
 
 // Request
 
 pub fn req() {
-    // TODO #4545: Implement test
+    // Generated from Python implementation (Parsec v2.6.0+dev)
+    // Content:
+    //   cmd: "message_get"
+    //   offset: 8
+    let raw = hex!("82a3636d64ab6d6573736167655f676574a66f666673657408");
+
+    let req = authenticated_cmds::message_get::Req { offset: 8 };
+
+    let expected = authenticated_cmds::AnyCmdReq::MessageGet(req);
+
+    let data = authenticated_cmds::AnyCmdReq::load(&raw).unwrap();
+
+    assert_eq!(data, expected);
+
+    let authenticated_cmds::AnyCmdReq::MessageGet(req2) = data else {
+        unreachable!()
+    };
+
+    let raw2 = req2.dump().unwrap();
+
+    let data2 = authenticated_cmds::AnyCmdReq::load(&raw2).unwrap();
+
+    assert_eq!(data2, expected);
```

Comment:
Consider re-using the `req` test from v3 (seems identical). Here's an example:
https://github.com/Scille/parsec-cloud/blob/0566f6e68cb953ee0385dcda64fef203cc05fe3e/libparsec/crates/protocol/tests/authenticated_cmds/v4/vlob_read.rs#L7-L13

Reply:
They're indeed identical, but I can't reuse-it because it would cause a compilation error for the `rep_ok` which Response don't share the same fields

Reply:
Ouch... this seems like something we'll probably come across quite often. 

I suppose it is because of the `use super::authenticated` in `v3/message_get.rs`. Maybe there is a way to force the `compat` module to use the `Rep` from `v3` :thinking:

Reply:
It's because of the `use super::authenticated`